cli/api.js

```diff
@@ -0,0 +1,112 @@
+const fs = require('fs');
+const os = require('os');
+const npmName = () => Promise.resolve(); //require('npm-name');
```

Comment:
🤔

Reply:
I hoped that this PR would have been merged sooner: https://github.com/dominictarr/rc/pull/94

Reply:
fixed